### ISSUE TITLE
Update Project Structure diagram in the documentation

### ISSUE THREAD

**Issue by h-mayorquin** [catalystneuro/neuroconv#1035]
The current one is out of date I think:

https://neuroconv.readthedocs.io/en/main/_images/neuroconv_uml.svg

**Comment by CodyCBakerPhD** [comment]
Yes, that was made using a custom UML diagram -> image file software, I much prefer the integrated Mermaid approach we have been using in extensions as of late
